data/coalition outfits.txt

```diff
@@ -18,7 +18,8 @@ outfit "Small Collector Module"
 	"outfit space" -8
 	"solar collection" .8
 	"heat generation" .4
-	description "Solar power is the mainstay of the Coalition's merchant marine. The design of these panels has been refined over the millennia, growing ever more efficient."
+	"ionization resistance" 1
```

Comment:
You meant `"ion resistance"` - ionization resistance is nothing special (note that there is no tooltip for this attribute of yours if you hover it in the Outfitter screen).

Reply:
Ion resistance fixed and committed.  Shield and hull buffs allow a stealth buff to coalition stock ships and allow greater flexibility in loadout on other ships as a reward for unlocking coalition license. This allows marginal ships like Leviathan and Falcon greater utility and makes human ships useful further into the late game. Lore-wise, I don't see a problem with aliens solving a problem that humans haven't seen the need to address.

Reply:
Hull + shield on outfits is a bit of a dead horse around here. It's something that MZ (the game's creator/primary author) has explicitly said several times he doesn't want used in the main game, as he wants to gradually create a progression of ship power, and use hull & shield as a way of obsoleting lower tier ships. It's probably best kept in a mod so that individual players can choose whether to use it or not, rather than submitted to the main branch.